deploy/python/infer.py

```diff
@@ -27,7 +25,7 @@
 from paddle.inference import create_predictor
 
 from benchmark_utils import PaddleInferBenchmark
-from preprocess import preprocess, Resize, NormalizeImage, Permute, PadStride
```

Comment:
Thank you for your modification,  The `Resize .etc` import here cannot be deleted.
```
Traceback (most recent call last):
  File "deploy/python/infer.py", line 632, in <module>
    main()
  File "deploy/python/infer.py", line 599, in main
    predict_image(detector, img_list, FLAGS.batch_size)
  File "deploy/python/infer.py", line 511, in predict_image
    results = detector.predict(batch_image_list, FLAGS.threshold)
  File "deploy/python/infer.py", line 133, in predict
    inputs = self.preprocess(image_list)
  File "deploy/python/infer.py", line 96, in preprocess
    preprocess_ops.append(eval(op_type)(**new_op_info))
  File "<string>", line 1, in <module>
NameError: name 'Resize' is not defined
```
You can test `deploy/python/infer`:
```
python tools/export_model.py -c configs/yolov3/yolov3_darknet53_270e_coco.yml --output_dir=./inference_model -o weights=https://paddledet.bj.bcebos.com/models/yolov3_darknet53_270e_coco.pdparams

python deploy/python/infer.py --model_dir=./inference_model/yolov3_darknet53_270e_coco --image_file=./demo/road554.png --device=GPU
```